src/config/keys.ts

```diff
@@ -0,0 +1,3 @@
+export default {
+    mongoURI: "mongodb+srv://ojtclass:topsecret@cluster0.neusy.mongodb.net/myFirstDatabase?retryWrites=true&w=majority"
```

Comment:
Is this for development? production? Basically, keys are not something we would upload to github, so how should we address this during the development process?

I personally would have a dev file with dev keys, a prod file with prod keys, and the keys file would be something like this: 
`if (process.env.NODE_ENV === 'production') { 
    module.exports = require('./prod')
} else {
    module.exports = require('./dev')
}`
but I'm not sure if this is at all relevant to our project

Reply:
Thank you very much orly!! I forgot to put the keys file in .gitignore:) 
Now I looked in nest.js docs to see what they recommend: https://docs.nestjs.com/techniques/configuration. 
I need to do some research and find the best solution for as for the moment. Implementing the ConfigModule. 
So, I will only add the current file to .gitignore, so everyone can work on what we have, And add the ConfigModule later today. 
Will send a new PR with changes for now.